lib/typescript/render/components/Media/index.module.scss

```diff
@@ -0,0 +1,16 @@
+.mediaImage {
```

Comment:
I've used Google's rich cards size for the media image height (https://developers.google.com/business-communications/business-messages/guides/build/send#card_height)